rasa/utils/io.py

```diff
@@ -186,22 +188,24 @@ def json_unpickle(file_name: Union[Text, Path]) -> Any:
     jsonpickle_numpy.register_handlers()
 
     file_content = rasa.shared.utils.io.read_file(file_name)
-    return jsonpickle.loads(file_content)
+    return jsonpickle.loads(file_content, keys=keys)
 
 
-def json_pickle(file_name: Union[Text, Path], obj: Any) -> None:
+def json_pickle(file_name: Union[Text, Path], obj: Any, keys: bool = False) -> None:
     """Pickle an object to a file using json.
 
     Args:
         file_name: the file to store the object to
         obj: the object to store
+        keys: If set to `True` then jsonpickle will encode non-string dictionary keys
```

Comment:
optional: Is a name like `encode_non_string_keys` more precise than? Just `keys` seems kind of vague

Reply:
👍